File: Corredor/Digitales/C#/C#/Profesor.cs
namespace EscuelaMusica
{
    public class Profesor
    {
        public string Nombre { get; set; }
        public string Instrumento { get; set; }
        public double Salario { get; set; }

        public Profesor(string nombre, string instrumento, double salario)
        {
            Nombre = nombre;
            Instrumento = instrumento;
            Salario = salario;
        }

        public void Enseñar()
        {
            System.Console.WriteLine($"{Nombre} enseña a tocar {Instrumento}.");
        }
    }
}

File: Corredor/Digitales/Go/profesor.go
package main

import "fmt"

type Profesor struct {
	nombre      string
	instrumento string
	salario     float64
}

func NuevoProfesor(nombre, instrumento string, salario float64) Profesor {
	return Profesor{nombre, instrumento, salario}
}

func (p Profesor) Enseñar() {
	fmt.Printf("%s enseña a tocar %s.\n", p.nombre, p.instrumento)
}